runtime/src/bank/tests.rs

```diff
@@ -1115,6 +1115,123 @@ fn test_distribute_rent_to_validators_overflow() {
     }
 }
 
+#[test]
+fn test_distribute_rent_to_validators_rent_paying() {
+    solana_logger::setup();
+
+    const RENT: u64 = 55;
```

Comment:
this is the individual rent due, i guess?

Reply:
Yep. Can name better.